packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -47,8 +48,8 @@ const styles: ComponentStyles<IMSFTProgressClassNameContract, IDesignSystem> = (
                 position: "absolute",
                 display: "inline-block",
                 background: brandColor,
-                width: "5px",
-                height: "5px",
+                width: toPx(designSystem.designUnit),

```

Comment:
Making sure these values were supposed to change from the prior values (i.e. they are all now `4px`)

Reply:
We're going to do a design review, but 5px doesn't really align with how elements are sized in the MSFT design language so I snapped them down a px.

Reply:
Cool